code/modules/mob/inventory.dm

```diff
@@ -38,7 +38,6 @@
 //This is an UNSAFE proc. It merely handles the actual job of equipping. All the checks on whether you can or can't eqip need to be done before! Use mob_can_equip() for that task.
 //In most cases you will want to use equip_to_slot_if_possible()
 /mob/proc/equip_to_slot(obj/item/W as obj, slot)
-	W.on_slotmove(src)

```

Comment:
That will screw with the deadman trigger.
Please continue to track it down the call stack and see if the issue with the shoes can be fixed some other way.

Reply:
I did tests without this proc here, and the deadman switch did work just fine, even when you equipped the signaler it into your pocket.

Reply:
Done.